xml/admin_ceph_cephfs.xml

```diff
@@ -601,7 +601,14 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
 </screen>
    <para>
     After you enable snapshots, all directories in the &cephfs; will have a
-    special <filename>.snap</filename> subdirectory.
+    special <filename>.snap</filename> subdirectory. Note that this is a virtual subdirectory.
+    In other words this directory does not appear in the directory listing of the parent directory,
+    but the name <filename>.snap</filename> can not be used as a file or directory name.
+    To access the <filename>.snap</filename> directory one has to explicitly access it, for exmaple
```

Comment:
```suggestion
    special <filename>.snap</filename> subdirectory.
  </para>
    <note>
      <para>
        This is a virtual subdirectory that does not appear in the directory listing
        of the parent directory, but the name <filename>.snap</filename> can not
        be used as a file or directory name. To access the <filename>.snap</filename>
        directory one has to explicitly access it, for example:
      </para>

Reply:
I am trying to make a suggestion concerning a suggestion here, and I'm not sure it worked. Please let me know if it needs clarification.